### PR TITLE
Add documentation on variable filtering in Hooks

### PR DESCRIPTION
**Pull request by mathieuleclaire**

### CODE REVIEW

File: openmole/bin/org.openmole.site/jvm/src/main/scalatex/openmole/documentation/language/Hook.scalatex [hunk 1]
```diff
@@ -197,7 +197,24 @@ You can think of the @code{DisplayHook} as an OpenMOLE equivalent to Scala's @co
...
+You can use the following notation in the hook function {@code{hook(yourHookHere, values = Seq(i,j))}}
+ 
+@br
+
+@hl.openmole("""
+  val i = Val[Int]
+  val j = Val[Int]
+  val k = Var[Double]
 
+  val task = ATask() set (
```
Comment (romainreuillon) [inline openmole/bin/org.openmole.site/jvm/src/main/scalatex/openmole/documentation/language/Hook.scalatex:211]: ATask() => EmptyTask()

**Review by romainreuillon — CHANGES_REQUESTED** [review]
The merge request should be open against master.

File: openmole/bin/org.openmole.site/jvm/src/main/scalatex/openmole/documentation/language/Hook.scalatex (hunk 1 above)
Comment (mathieuleclaire) [reply openmole/bin/org.openmole.site/jvm/src/main/scalatex/openmole/documentation/language/Hook.scalatex:211]: Well, it was pseudo-code, but I forgot it was compiled.

Comment (romainreuillon) [reply openmole/bin/org.openmole.site/jvm/src/main/scalatex/openmole/documentation/language/Hook.scalatex:211]: Could you fix the task type as well ?

**Pull request merged**